src/services/user/services/AdminUsers.js

```diff
@@ -18,17 +19,24 @@ const getRoles = () => roleModel.find()
 	.exec();
 
 
-const getAllUsers = (ref, schoolId, role, clientQuery) => {
+const getAllUsers = async (ref, schoolId, schoolYearId, role, clientQuery = {}) => {
 	const query = {
 		schoolId,
-		roles: role.toString(),
-		$sort: clientQuery.$sort,
-		$select: ['firstName', 'lastName', 'email', 'createdAt', 'importHash', 'birthday'],
+		roles: role,
+		schoolYearId,
+		consentStatus: clientQuery.consentStatus,
+		sort: clientQuery.$sort || clientQuery.sort,
+		select: ['consentStatus', 'consent', 'classes', 'firstName', 'lastName', 'email', 'createdAt', 'importHash', 'birthday'],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 129. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5276619)